### PR TITLE
Add link to FCS preview versions

### PR DESCRIPTION
**Pull request by nojaf**
Correct small typo and add a link to the versions of the prerelease FCS package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Even if you find a single-character typo, we're happy to take the change! Althou
 
 ## Per-build NuGet packages
 
-Per-build verions of our NuGet packages are available via this URL: `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json `
+Per-build [versions](https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-tools&view=versions&package=FSharp.Compiler.Service&protocolType=NuGet) of our NuGet packages are available via this URL: `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json `
```
Comment (baronfel) [inline README.md:65]: Pre* build?

Comment (baronfel) [reply README.md:65]: Though maybe this should be more like 'prerelease' versions?

Comment (vzarytovskii) [reply README.md:65]: I think "per-build" makes sense here, as in "one package per build (of main)"?

Comment (baronfel) [reply README.md:65]: I was under the impression there are prereleases from a few different branches (possibly release/FCS as well)? This is based on version number change propagation as well as pulling FCS 40 prereleases in FSAC for a bit and seeing wildly different API surface areas day to day 😂

**Review by KevinRansom — APPROVED** [review]

**Pull request merged**